addons/medical_feedback/script_component.hpp

```diff
@@ -42,3 +42,16 @@
 #define FX_PAIN_PULSATING_BLUR 1
 #define FX_PAIN_CHROMATIC_ABERRATION 2
 #define FX_PAIN_ONLY_BASE 3
+
+#define FX_BLOODVOLUME_COLOR_CORRECTION 0
+#define FX_BLOODVOLUME_ICON 1
+#define FX_BLOODVOLUME_BOTH 2
+
+#define ICON_BLOODVOLUME_IDX_MIN 0
+#define ICON_BLOODVOLUME_IDX_MAX 6
+
+#define ICON_BLOODVOLUME_PATH(num)    format [QPATHTOF(data\bloodVolume_%1.paa), num]
+#define ICON_BLOODVOLUME_COLOR_WHITE  [1, 1, 1, 1]
+#define ICON_BLOODVOLUME_COLOR_ORANGE [1, 0.6, 0, 1]
+#define ICON_BLOODVOLUME_COLOR_RED    [0.8, 0.2, 0, 1]
+#define ICON_BLOODVOLUME_COLOR_NONE   [0, 0, 0, 0]
```

Comment:
Annoying white space. We don't do this type of formatting, as it leads to wall of pink whenever a macro or whatever with a longer name is added. Personally I'd also delete the space after commas for pure number arrays, but that is inconsistent/not generally accepted.

Reply:
Removed whitespaces after macro name